sds_data_manager/lambda_code/SDSCode/batch_starter.py

```diff
@@ -2,7 +2,6 @@
 
```

Comment:
I'm trying to think if anything here would be any different for L3. I'm just brainstorming here:
Would we need to pass in the binning mechanism here (abrate vs HEALPIX etc) in the command?

Reply:
I think that is a good question, and we would need to update the command sent to the container. Where that command comes from is TBD though if we do need something else.